### PR TITLE
Feedback-request + Feedback

### PR DESCRIPTION
**Pull request by Farleo**

### CODE REVIEW

File: src/main/java/ita/softserve/course_evaluation/CourseEvaluationApplication.java [hunk 1]
```diff
@@ -10,4 +15,14 @@ public static void main(String[] args) {
 		SpringApplication.run(CourseEvaluationApplication.class, args);
 	}
 	
+	@Bean
+	public ModelMapper modelMapper() {
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/CourseEvaluationApplication.java:19]: I suggest creating a configuration class for this

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	private final FeedbackService feedbackService;
+	
+	public FeedbackController(FeedbackService feedbackService) {
+		this.feedbackService = feedbackService;
+	}
+	
+	@PostMapping
+	@ResponseStatus(HttpStatus.OK)
+	public void createFeedback(@RequestBody FeedbackDto dto) {
+		feedbackService.create(dto);
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java:30]: Usually create method returns created object

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	
+	@GetMapping("/{id}")
+	public ResponseEntity<FeedbackDto> getFeedback(@PathVariable Long id) {
+		return ResponseEntity.status(HttpStatus.OK)
+				       .body(feedbackService.getFeedbackById(id));
+	}
+	
+	@PutMapping("/{id}")
+	@ResponseStatus(HttpStatus.OK)
+	public void editFeedback(@RequestBody FeedbackDto dto, @PathVariable Long id) {
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java:41]: Same as with create

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestController.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	private final FeedbackRequestService feedbackRequestService;
+	
+	public FeedbackRequestController(FeedbackRequestService feedbackRequestService) {
+		this.feedbackRequestService = feedbackRequestService;
+	}
+	
+	@PostMapping
+	@ResponseStatus(HttpStatus.OK)
+	public void createFeedbackRequest(@RequestBody FeedbackRequestDto dto) {
+		feedbackRequestService.create(dto);
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestController.java:30]: same here, create

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java (hunk 2 above)
Comment (Farleo) [reply src/main/java/ita/softserve/course_evaluation/controller/FeedbackController.java]: ok

File: src/main/java/ita/softserve/course_evaluation/CourseEvaluationApplication.java (hunk 1 above)
Comment (Farleo) [reply src/main/java/ita/softserve/course_evaluation/CourseEvaluationApplication.java]: i moved it to -> /configuration/ApplicationConfig.java

**Review by Forfend — APPROVED** [review]

**Pull request merged**